docs/index.md

```diff
@@ -21,5 +21,8 @@ You can follow our getting started guide, which will provide instructions on how
 - [Webhooks](usage/webhooks.md)
```

Comment:
Are we updating the getting started guide index accordingly?

Reply:
I took a look and don't think anything we're doing here effects the getting started guide.